.github/workflows/smoke-test.yml

```
@@ -28,7 +28,7 @@ jobs:
           android: true         # 8.7GiB
           dotnet: true          # 1.6GiB
           docker-images: true   # 3.2GiB
-          haskell: false        # no: little/no benefit
+          haskell: true         # ?
```

```suggestion
          haskell: true         # 5.4GiB
```
presumably the same? or maybe just keep the numbers in one place.